_posts/0000-03-01-maalbash.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+![bad-luck-brian](https://cloud.githubusercontent.com/assets/16547949/25400743/8ae447a2-29c1-11e7-9839-5438d8fcda7b.jpg)
+{: .center}
+
+When your PC BSOD's during class
```

Comment:
Or when your screen starts flickering like in Stranger Things 😱 😭

Reply:
No spoilers please